core/arch/arm/plat-zynqmp/main.c

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) 2016, Xilinx Inc.
+ * Copyright (c) 2014, STMicroelectronics International N.V.
```

Comment:
Copy/paste? This is a new file for zynqmp and even though you might have used the ST platform as a reference I don't think you should include them in the copyright in this particular file.